### DISCUSSION TITLE
Debugger info

### DISCUSSION THREAD

**Discussion by alexpamp** [XusinboyBekchanov/MyFbFramework#89] [General]
Hi, I'm trying the debugger in the MDIForm example but it doesn't stop at breakpoints in the MDI form. Instead in the other modules it stops normally. Attached is the screenshot. Using the debugger gdb stops normally. Did I forget some settings? Many thanks. Alessandro
![Debugger](https://user-images.githubusercontent.com/123064597/214819067-3e358831-f69f-4f57-a6ef-2c5e9423bba4.jpg)

**Comment by XusinboyBekchanov** [comment]
Fixed: Debugging Main File with Integrated IDE Debugger:
https://github.com/XusinboyBekchanov/VisualFBEditor/commit/536e49ab997332d331687181679ca7d9478a15df

**Reply by alexpamp** [reply]
Perfect! Works! Great!
